source/options.tsx

```diff
@@ -74,6 +75,20 @@ function expandTokenSection(): void {
 	select('details#token')!.open = true;
 }
 
+async function updateStorageUsage(area: 'sync' | 'local'): Promise<void> {
+	const storage = browser.storage[area];
+	const used = await storage.getBytesInUse();
+	// @ts-expect-error Type issue, we should use @types/webextension-polyfill
+	const available = storage.QUOTA_BYTES - used;
+	for (const output of select.all(`.storage-${area}`)) {
+		output.textContent = available < 1000
+			? 'FULL!'
+			: (available < 100_000
+				? `Only ${prettyBytes(available)} available`
+				: `${prettyBytes(used)} used`);
```

Comment:
Logic: I don't need to know I have 10MB available, it captures too much attention.

I tried with a `progress` element but most of the times it looks so empty it's unclear.